prow/plugins.yaml

```diff
@@ -5,10 +5,9 @@
 triggers:
 - repos:
   - istio
-  - istio-releases/pipeline
+  - istio-ecosystem
+  - istio-releases
   trusted_org: istio
-- repos:
-  - istio-ecosystem/authservice
```

Comment:
Note that this is a breaking change: Previously user trust in `istio-ecosystem/authservice` was based on membership in `istio-ecosystem`. Now it will be base on membership in `istio`.

Reply:
This doesn't remove the istio-ecosystem trust. It means we trust anyone in the istio-ecosystem OR istio org.

https://github.com/kubernetes/test-infra/blob/2c53c2d7e3e07a95005d04ca7e36dd53c438582e/prow/plugins.yaml#L5-L13

Reply:
And more specifically: https://github.com/kubernetes/test-infra/blob/master/prow/plugins/trigger/trigger.go#L168-L178